pkg/proxy/handlers/spacelister_get.go

```diff
@@ -0,0 +1,173 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sort"
+	"time"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/registration-service/pkg/context"
+	"github.com/codeready-toolchain/registration-service/pkg/metrics"
+	commonproxy "github.com/codeready-toolchain/toolchain-common/pkg/proxy"
+	"github.com/codeready-toolchain/toolchain-common/pkg/spacebinding"
+	"github.com/labstack/echo/v4"
+	errs "github.com/pkg/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/selection"
+)
+
+func (s *SpaceLister) HandleSpaceGetRequest(ctx echo.Context) error {
```

Comment:
I've converted all the methods to functions in those two files, and I'm passing the spaceLister as a parameter.
Please see https://github.com/codeready-toolchain/registration-service/pull/371/commits/192aef489335f6cf21784b753fe8138e4c542576

I can also revert this split if it looks cleaner only to me. The main point was to ease the file navigation, I was a bit lost scrolling up and down searching for things, since the `spacelister.go` file was getting bigger and bigger.